psh/test-mkdir.py

```diff
@@ -0,0 +1,143 @@
+import random
+import string
+
+from collections import namedtuple
+
+File = namedtuple('File', ['name', 'owner', 'is_dir'])
+
+def ls(p, dir=''):
+    files = []
+
+    p.sendline(f'ls -la {dir}')
+    p.expect_exact('ls -la')
+    if dir:
+        p.expect_exact(f' {dir}')
+
+    p.expect_exact('\n')
+
+    while True:
+        idx = p.expect(['\(psh\)\% ', '(.*)(\r+)\n'])
```

Comment:
```suggestion
        idx = p.expect([r'\(psh\)\% ', '(.*)(\r+)\n'])
```
I think it should be `raw string`.